lazy_theta_star_b/package.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>lazy_theta_star_b</name>
+  <version>0.0.1</version>
+  <description>TODO: Package description</description>
+  <maintainer email="Anshu-man567@todo.todo">anshu-man</maintainer>
```

Comment:
Please fill this in

Reply:
Is your's and Carlos' IDs to be filled??

Reply:
Yours and mine,  I suppose